### ISSUE TITLE
Test name is empty

### ISSUE THREAD

**Issue by jscriptcoder** [dhershman1/tap-junit#8]
Hi there,

There seems to be a problem with the name of the tests. After upgrading the version, I realised that I no longer have the test name in the xml.

Looking at the source code I saw that we're passing an object to ```newTest``` that expects ```testName``` as property, but then we're calling this function passing ```name``` instead
```javasceript
testCase = newTest({ name });
```
and
```javascript
tap.on('test', res => {
  testCase = newTest(res);
});
```
where ```res``` has a property ```name```. See [test.js](https://github.com/scottcorgan/tap-out/blob/master/lib/test.js) in tap-out.

Sending a pull request shortly.

Fran

**Comment by jscriptcoder** [comment]
Pull request [done](https://github.com/dhershman1/tap-junit/pull/9). Please, have a look.

Thanks,
Fran

**Comment by dhershman1** [comment]
PR Merged in, and npm version 1.2.10 published. Thanks for the hard work!
